scripts/ci/images/ci_start_arm_instance_and_connect_to_docker.sh

```diff
@@ -20,7 +20,7 @@ SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
 WORKING_DIR="/tmp/armdocker"
 INSTANCE_INFO="${WORKING_DIR}/instance_info.json"
 ARM_AMI="ami-0e43196369d299715"  # AMI ID of latest arm-docker-ami-v*
-INSTANCE_TYPE="m7g.2xlarge"  # m7g.2xlarge -> 8 vCPUS 32 GB RAM
+INSTANCE_TYPE="m7g.xlarge"  # m7g.2xlarge -> 8 vCPUS 32 GB RAM
```

Comment:
Comment is not matching anymore
```suggestion
INSTANCE_TYPE="m7g.xlarge"  # m7g.xlarge -> 4 vCPUS 16 GB RAM
```

Reply:
Right